src/halmos/sevm.py

```diff
@@ -2194,24 +2365,24 @@ def gen_nested_ite(curr: int) -> BitVecRef:
         # If(idx == 0, Extract(255, 248, w), If(idx == 1, Extract(247, 240, w), ..., If(idx == 31, Extract(7, 0, w), 0)...))
         return ZeroExt(248, gen_nested_ite(0))
 
-    def run(self, ex0: Exec) -> Tuple[List[Exec], Steps]:
+    def run(self, ex0: Exec) -> Tuple[List[Exec], Steps, HalmosLogs]:
         out: List[Exec] = []
-        logs = Logs()
+        logs = HalmosLogs()
         steps: Steps = {}
         step_id: int = 0
 
         stack: List[Tuple[Exec, int]] = [(ex0, 0)]
         while stack:
             try:
-                if (
-                    "max_width" in self.options
-                    and len(out) >= self.options["max_width"]
-                ):
+                if len(out) >= self.options.get("max_width", 2**64):
```

Comment:
2**64 is nice.  can we just have it as the default value of max_width, so that it can be also displayed in the --help message?